bigtable/client/rpc_backoff_policy.cc

```diff
@@ -55,5 +55,28 @@ std::chrono::milliseconds ExponentialBackoffPolicy::on_completion(
   return delay;
 }
 
+std::unique_ptr<RPCBackoffPolicy> RandomizedBackoffPolicy::clone() const {
+  return std::unique_ptr<RPCBackoffPolicy>(new RandomizedBackoffPolicy(*this));
+}
+
+void RandomizedBackoffPolicy::setup(grpc::ClientContext& /*unused*/) const {}
+
+std::chrono::milliseconds RandomizedBackoffPolicy::on_completion(
+    grpc::Status const& status) {
+  using namespace std::chrono;
+  auto delay = duration_cast<milliseconds>(current_delay_);
+  std::random_device rd;
+  std::mt19937 rng_generator(rd());
```

Comment:
Please call the variable `rng` or `generator`, right now I read "random number generator generator" in my head.

Reply:
Done.